config/var_development.env

```diff
@@ -0,0 +1,6 @@
+DB_CONNECT_STRING=pgsql:host=apiservice.cicyc5fazypj.us-east-1.rds.amazonaws.com;dbname=nypl
```

Comment:
is this DB_CONNECT_STRING safe to store here?

Reply:
Yeah I think so. A db host is useless without a password and the security group prevents hitting it outside the VPC.